test/personalisedcoverdownload.spec.js

```diff
@@ -0,0 +1,68 @@
+let expect = chai.expect;
+
+// load in component(s) to be tested
+let PersonalisedCoverDownload = require('../assets/js/components/PersonalisedCoverDownload');
+
+describe('A PersonalisedCoverDownload Component', () => {
+  'use strict';
+
+  let $elm = document.querySelector('[data-behaviour="PersonalisedCoverDownload"]');
+  let personalisedCoverDownload;
+
+  beforeEach(() => {
+    personalisedCoverDownload = new PersonalisedCoverDownload($elm, window, window.document);
+  });
+
+  it('exists', () => {
+    expect(personalisedCoverDownload).to.exist;
+  });
+
+  describe('toggleButtons()', () => {
+    let a4ButtonCollection;
+    let letterButtonCollection;
+
+    before(() => {
+      a4ButtonCollection = $elm.querySelector('.button-collection--a4');
+      letterButtonCollection = $elm.querySelector('.button-collection--usletter');
+    });
+
+    context('default buttons', () => {
+      it('the A4 download menu is not hidden', () => {
```

Comment:
In previous tests we have made these so they read like sentences so for example:

"When A4 has been selected"

"the A4 download menu is not hidden"

"and the US letter download menu is hidden"

Do you see what I mean or are we changing the way we are doing our tests?

Reply:
Have reworded the descriptions